### PR TITLE
Scoping limitations of EDR Block Mode

### PR DESCRIPTION
**Pull request by anvascon**
Added info that customers cannot currently scope EDR in Block mode.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/edr-in-block-mode.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ The following image shows an instance of unwanted software that was detected and
 3. Scroll down, and then turn on **Enable EDR in block mode**.
 
 > [!NOTE]
-> EDR in block mode can be turned on only in the <a href="https://go.microsoft.com/fwlink/p/?linkid=2077139" target="_blank">Microsoft 365 Defender portal</a> or the former Microsoft Defender Security Center ([https://securitycenter.windows.com](https://securitycenter.windows.com)). You cannot use registry keys, Microsoft Intune, or Group Policy to enable or disable EDR in block mode.
+> EDR in block mode can be turned on only in the <a href="https://go.microsoft.com/fwlink/p/?linkid=2077139" target="_blank">Microsoft 365 Defender portal</a> or the former Microsoft Defender Security Center ([https://securitycenter.windows.com](https://securitycenter.windows.com)), and it is only a tenant wide setting (not possible to target it at device groups or users). You cannot use registry keys, Microsoft Intune, or Group Policy to enable or disable EDR in block mode.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/edr-in-block-mode.md]:
```suggestion
> EDR in block mode can be turned on only in the <a href="https://go.microsoft.com/fwlink/p/?linkid=2077139" target="_blank">Microsoft 365 Defender portal</a> or the former Microsoft Defender Security Center ([https://securitycenter.windows.com](https://securitycenter.windows.com)), and it is only a tenant wide setting (it is not possible to target it at device groups or users). You cannot use registry keys, Microsoft Intune, or Group Policy to enable or disable EDR in block mode.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @anvascon, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @anvascon @JohanFreelancer9

**Pull request merged**